pkgs/sketch_pad/lib/widgets.dart

```diff
@@ -107,6 +107,9 @@ class RunButton extends StatelessWidget {
               RoundedRectangleBorder(borderRadius: BorderRadius.circular(4.0))),
           backgroundColor: MaterialStateProperty.resolveWith(
             (states) {
+              if (onPressed == null) {
+                return Colors.transparent;
+              }
               return const Color(0xFF168afd);
```

Comment:
This ends up looking a bit awkward since the button is still there. What do you think about using the same background color, but with a reduced opacity?

Also more standard way to do this might be to use the closure's passed in `states` variable to check if the button is disabled. That way it would be consistent with any other states we might customize in the future.

So together, that could look something like:

```suggestion
              const buttonColor = Color(0xFF168afd);
              if (states.contains(MaterialState.disabled)) {
                return buttonColor.withOpacity(0.4);
              }
              return buttonColor;
```

Reply:
Hey thanks, for the suggestion. 
You are right ; transition is better. So i just copy/past your suggestion and make a new PR. Can you review it @parlough ?